mapeditor/inspector/inspector.cpp

```diff
@@ -540,7 +540,7 @@ void Inspector::setProperty(CGArtifact * o, const QString & key, const QVariant
 	{
 		for(auto spell : VLC->spellh->objects)
 		{
-			if(spell->getJsonKey() == value.toString().toStdString())
+			if(spell->getNameTranslated() == value.toString().toStdString())
```

Comment:
What if there are two spells with same name?

Reply:
I'm working on PR with proper way of handling objects via id